### PR TITLE
Support table, column and index comments

### PR DESCRIPTION
**Pull request by timruffles**

### CODE REVIEW

File: goose/00001_initial.sql [hunk 1]
```diff
@@ -14,12 +14,12 @@ CREATE TABLE IF NOT EXISTS companies (
 
 CREATE TABLE IF NOT EXISTS persons (
   person_id         INT,
-  last_name         VARCHAR(255),
-  first_name        VARCHAR(255),
+  last_name         VARCHAR(100),
+  first_name        VARCHAR(100),
```
Comment (timruffles) [inline goose/00001_initial.sql:18]: this wouldn't work on mysql 5.6, where there's a 756 byte limit on indexes (utf8mb4, so 255 * 4 * 2)

File: format/format.go [hunk 2]
```diff
@@ -8,10 +8,6 @@ import (
 	"github.com/olekukonko/tablewriter"
 )
 
-func makeTitle(s string) string {
```
Comment (ntindall) [inline format/format.go:11]: thanks

File: fixtures/expected1.md [hunk 3]
```diff
@@ -1,90 +1,93 @@
...
-| KEY NAME  | UNIQUE |    COLUMNS     |
-|-----------|--------|----------------|
-| `PRIMARY` | `true` | `(company_id)` |
+| KEY NAME  | UNIQUE |    COLUMNS     | COMMENT |
+|-----------|--------|----------------|---------|
+| `PRIMARY` | `true` | `(company_id)` |         |
 
 ## employees
+
+this is a table comment
```
Comment (ntindall) [inline fixtures/expected1.md:40]: 👍

**Review by ntindall — APPROVED** [review]
🌱 LGTM - thanks for this.

I think an improvement would be to omit the COMMENT column if there are no comments for any of the columns / indexes, but we can always add this in later.

**Pull request merged**